src/main/java/uk/nhs/digital/nhsconnect/lab/results/outbound/queue/ObjectSerializer.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.digital.nhsconnect.lab.results.outbound.queue;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.outbound.fhir.FhirParser;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
Should we do it in this way?
```suggestion
@RequiredArgsConstructor(onConstructor_ = @Autowired)
```

Reply:
Resolving, because already has been mentioned by Andrew